README.md

```diff
@@ -75,29 +85,61 @@ Flags:
+| `build-workflow-input` | Expects key-value pairs like `key=value` to match against [inputs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs) for GitHub Actions `workflow_dispatch` triggers. |
+
+## Verification for GitHub generators
+
+### Artifacts
+
+To verify an artifact, run the following command:
 
 ```bash
-$ go run ./cli/slsa-verifier -provenance-path ~/Downloads/slsa-verifier-linux-amd64.intoto.jsonl --source-uri github.com/slsa-framework/slsa-verifier --source-tag v1.3.0 ~/Downloads/slsa-verifier-linux-amd64 
+$ slsa-verifier verify-artifact slsa-test-linux-amd64 \
+  --provenance-path slsa-test-linux-amd64.intoto.jsonl \
+  --source-uri github.com/slsa-framework/slsa-test \
+  --source-tag v1.0.3
 Verified signature against tlog entry index 3189970 at URL: https://rekor.sigstore.dev/api/v1/log/entries/206071d5ca7a2346e4db4dcb19a648c7f13b4957e655f4382b735894059bd199
 Verified build using builder https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v1.2.0 at commit 5bb13ef508b2b8ded49f9264d7712f1316830d10
 PASSED: Verified SLSA provenance
 ```
 
 The verified in-toto statement may be written to stdout with the `--print-provenance` flag to pipe into policy engines.
 
-### Options Details
 
-The following options are supported for [SLSA GitHub builders and generators](https://github.com/slsa-framework/slsa-github-generator#generation-of-provenance):
+### Containers
+This is WIP and currently not supported.
 
-| Option | Description |
-| --- | ----------- |
-| `source-uri` | Expects a source, for e.g. `github.com/org/repo`. |
-| `source-branch` | Expects a `branch` like `main` or `dev`. Not supported for all GitHub Workflow triggers. |
-| `source-tag` | Expects a  `tag` like `v0.0.1`. Verifies exact tag used to create the binary. NSupported for new [tag](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.go.tag.main.config-ldflags-assets-tag.slsa3.yml#L5) and [release](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.go.release.main.config-ldflags-assets-tag.slsa3.yml) triggers. |
-| `source-versioned-tag` | Like `tag`, but verifies using semantic versioning. |
-| `build-workflow-input` | Expects key-value pairs like `key=value` to match against [inputs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs) for GitHub Actions `workflow_dispatch` triggers. |
+## Verification for Google Cloud Build
+
+### Artifacts
+This is WIP and currently not supported.
 
+### Containers
+To verify a contaimer image, run the following command:
+
+```bash
+$ IMAGE=us-west2-docker.pkg.dev/gosst-scare-sandbox/quickstart-docker-repo/quickstart-image:v39
+$ IMAGE=$(docker inspect --format='{{.RepoDigests}}' ${IMAGE} | cut -f1 -d ' ' | cut -d "[" -f2 | cut -d "]" -f1)
```

Comment:
i'll try this when the image is public but to avoid docker pull before docker inspect, maybe we can use `crane digest`?

Reply:
I'll provide the 2 options?
What you're saying is that we should not be pulling an image unless it's trusted, correct?

Reply:
I don't think it's *needed*, although i think it is better to verify before pulling, but only because i think this second command won't work unless there's a docker pull before it

Reply:
done.

Reply:
agreed. I only added the crane command. I wanted to avoid asking users to install another tool. But I think it also avoids asking users to rm the image from their local cache if verification fails.